package.json

```diff
@@ -37,7 +38,7 @@
     "watch:js": "run-p watch:js:*",
     "watch:js:webpack": "webpack --watch --mode development --devtool inline-source-map  --config ./webpack.config.js",
     "test": "run-s test:*",
-    "test:functional": " nyc --reporter=lcov --reporter=text mocha --timeout 15000 --require ignore-styles \"test/functional/**/*.test.js\"",
+    "test:functional": "c8 mocha --timeout 15000 --require ignore-styles \"test/functional/**/*.test.js\"",
```

Comment:
`c8` is a drop-in replacement for `nyc`. `nyc` fails with ESM on Node > 16.12. Src:
- https://github.com/istanbuljs/nyc/issues/659
- https://github.com/istanbuljs/nyc/issues/1287
- https://stackoverflow.com/a/68095739